modules/ACASBarcodeSELPreprocessor/src/client/ACASBarcodeSELPreprocessor.coffee

```diff
@@ -1,4 +1,4 @@
-class window.ACASBarcodeSELPreprocessor extends BasicFileValidateAndSaveController
+class ACASBarcodeSELPreprocessor extends BasicFileValidateAndSaveController
```

Comment:
`class window.ClassName` was changed in all cases to `class ClassName`  This is because the fat arrows changed meaning in coffeescript 2.  We updated to coffeescript to now support async/await function.  Coffeescript 2 does preserve the meaning of fat arrows but not when used like `class window.ClassName`  

See here for more details: https://coffeescript.org/#breaking-changes